src/RequestForm.js

```diff
@@ -1586,12 +1578,14 @@ class RequestForm extends React.Component {
                                     name="requester.barcode"
                                     placeholder={placeholder}
                                     aria-label={<FormattedMessage id="ui-requests.requester.barcode" />}
+                                    label={<FormattedMessage id="ui-requests.requester.barcode" />}
```

Comment:
Please omit `aria-label` now that this field properly has a `label`.

Reply:
Fixed